geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/MySqlJdbcDistributedTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.connectors.jdbc;
+
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Types;
+
+import org.junit.ClassRule;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.test.dunit.rules.MemberVM;
+import org.apache.geode.test.junit.categories.DistributedTest;
+import org.apache.geode.test.junit.rules.MySqlConnectionRule;
+import org.apache.geode.test.junit.rules.SqlDatabaseConnectionRule;
+
+/**
+ * End-to-end dunits for jdbc connector
+ */
+@Category(DistributedTest.class)
+public class MySqlJdbcDistributedTest extends JdbcDistributedTest {
+
+  private static final URL COMPOSE_RESOURCE_PATH =
+      MySqlJdbcDistributedTest.class.getResource("mysql.yml");
+
+  @ClassRule
+  public static transient SqlDatabaseConnectionRule dbRule = createConnectionRule();
+
+  private static SqlDatabaseConnectionRule createConnectionRule() {
+    try {
+      return new MySqlConnectionRule.Builder().file(COMPOSE_RESOURCE_PATH.getPath())
+          .serviceName("db").port(3306).database(DB_NAME).build();
+    } catch (IllegalStateException e) {
+      return null;
+    }
+  }
+
+  @Override
+  public Connection getConnection() throws SQLException {
+    return dbRule.getConnection();
+  }
+
+  @Override
+  public String getConnectionUrl() {
+    return dbRule == null ? null : dbRule.getConnectionUrl();
+  }
+
+  @Override
+  protected void createSupportedFieldsTable(Statement statement, String tableName)
+      throws SQLException {
+    statement.execute("CREATE TABLE " + tableName + " (id varchar(10) primary key not null, "
+        + "aboolean smallint, " + "abyte smallint, " + "ashort smallint, " + "anint int, "
+        + "along bigint, " + "aFloat float, " + "ADOUBLE double, " + "astring varchar(10), "
+        + "adate datetime, " + "anobject varchar(20), " + "abytearray blob(100), "
```

Comment:
I think the only reason you could not have this up in the super class is the "abytearray" column.
You should be able to make this column varbinary(100) and that would work for byte[] on both dbs.
We could try this out after you push this change if you do want to get this committed.

Reply:
SInce bytea is the postrgres equivalent of blob, which the original tests used, I went with testing those specific types. If we want to change the types tested or what they should map to, we can always modify them, though I was thinking that should be outside the scope of this change.